Documentation/build-master-role.md

```diff
@@ -0,0 +1,42 @@
+# Build Master role
+
+In the Mender team, we have a weekly rotative role known as "Build Master". This
+is a person in the team responsible for:
+
+* Monotiring the health of the Build System
+* Doing early investigation on issues
+* Report problems to the team
+
+## Mender QA rotation calendar
+
+Link: https://calendar.google.com/calendar/embed?src=northern.tech_m6aop2it00n2jnpiut2j40n39k%40group.calendar.google.com&ctz=Europe%2FOslo
+
+This is the Google calendar that shows who takes this role when.
+
+## Reponsabilities
+
+### Nightly Mender QA build
+
+Link: https://gitlab.com/Northern.tech/Mender/mender-qa/pipeline_schedules
```

Comment:
@merlin-northern has suggested to use proper markdown links here (i.e. `[]()` syntax). At least for the gitlab related ones, I like them to be explicit.

Are there other opinions out there?

Reply:
No strong opinions. We're not really rendering this, so it's more rich text than anything really